src/Paket.Core/Dependencies/RemoteDownload.fs

```diff
@@ -134,18 +134,24 @@ let downloadDependenciesFile(force,rootPath,groupName,parserF,remoteFile:ModuleR
         if exists then
             return parserF (File.ReadAllText(destination.FullName))
         else
-            let! result = lookupDocument(auth,url)
-
-            let text,depsFile =
-                match result with
-                | SuccessResponse text ->
-                        try
-                            text,parserF text
-                        with 
-                        | _ -> "",parserF ""
-                | NotFound
-                | UnknownError _ ->
-                    "",parserF ""
+            let! text,depsFile = async {
+                // TODO: Fixme, something is wrong on testcase #1341
+                if url <> "file://" then
```

Comment:
I think this was broken ever since... Something fishy is going on when paket tries to find the `paket.dependencies` file from a `http file:///C:/....` entry. This was noticed, because `HttpClient` no longer supports `file:///` and I could see in the debugger that this code tries to download `file://` which looks horribly wrong. But as this is not related to this PR I added this "hack" as I have no idea what this is supposed to do.